### PR TITLE
James whip

### PR DESCRIPTION
**Pull request by codemonkeman**

### CODE REVIEW

File: src/components/ViewParty/ViewParty.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React, { useState, useContext } from 'react';
+import { Table, Button, Accordion, Card } from 'react-bootstrap';
+import cookies from 'js-cookie';
+import './ViewParty.css'
+import { UserContext } from '../../UserContext'
+import axios from 'axios';
+
+//temporary metadata until we can pull it from the DB
+const tempPartyList = [
```
Comment (ThaddG) [inline src/components/ViewParty/ViewParty.js]: You can actually remove this list. ViewParty component will work better as a component that accepts props.

File: src/components/ViewParty/ViewParty.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      Name: "Detroit LAN",
+      Location: "Detroit",
+      Latitude: 42.331427,
+      Longitude: -83.0457538,
+      Date: "Thu Oct 22 2020 00:58:34 GMT-0400"
+    }
+  ]
+
+
+const ViewParty=()=>{
```
Comment (ThaddG) [inline src/components/ViewParty/ViewParty.js]: Change this to const ViewParty = (props) => {

File: src/components/ViewParty/ViewParty.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+    return(
+        <div>
+        {tempPartyList.map((p) => (
```
Comment (ThaddG) [inline src/components/ViewParty/ViewParty.js]: Remove this line but keep everything under it.

File: src/components/ViewParty/ViewParty.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+          <div 
+            style={{
+              padding: "10px 10px 5px",
+              borderBottom:"2px solid #0C0C0D",
+              backgroundColor: "#35373D"
+            }}>
+            <h4>{p.Name}</h4>
```
Comment (ThaddG) [inline src/components/ViewParty/ViewParty.js]: Change "p" to "props". Do the same for other instances of "p"

**Review by ThaddG — CHANGES_REQUESTED** [review]

File: src/components/ViewParty/ViewParty.js (hunk 1 above)
Comment (codemonkeman) [reply src/components/ViewParty/ViewParty.js]: Handled, got rid of the list.

File: src/components/ViewParty/ViewParty.js (hunk 2 above)
Comment (codemonkeman) [reply src/components/ViewParty/ViewParty.js]: added props

File: src/components/ViewParty/ViewParty.js (hunk 3 above)
Comment (codemonkeman) [reply src/components/ViewParty/ViewParty.js]: tempPartyList.map((p) => (
and accompanying {} removed

File: src/components/ViewParty/ViewParty.js (hunk 4 above)
Comment (codemonkeman) [reply src/components/ViewParty/ViewParty.js]: all instances of p changed to props

**Pull request merged**